### PR TITLE
Landing page setup table

### PR DESCRIPTION
**Pull request by kabelic**
hello bro, i did fetch and displayed the data.
another question are you able to see my UI when reviewing?

### CODE REVIEW

File: src/components/LinkComponent.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { LinkWrapper } from "./styles/Nav.styled"
+
+const LinkComponent = ({src, children}) => {
+    return (
+    <LinkWrapper to={src}>{children}</LinkWrapper>
```
Comment (yoloOnTheBattlefield) [inline src/components/LinkComponent.js:5]: indent

File: src/components/navComponent/NavComponent.js [hunk 2]
```diff
@@ -1,22 +1,21 @@
+import {SiNintendoswitch} from 'react-icons/si'
```
Comment (yoloOnTheBattlefield) [inline src/components/navComponent/NavComponent.js:1]: import { SiNintendoswitch } from 'react-icons/si'

File: src/components/navComponent/NavComponent.js [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
+import { LeftSide, Nav, RightSide, RightCornerIcon, LinkStyle, LinkWrapper } from '../styles/Nav.styled';
 
-const NavComponent = () => {
+const NavComponent = (props) => {
     return(
         <Nav>
             <LeftSide>
                 <Coins/>
-                <LinkStyle><Link style={{textDecoration: 'none', color:'white'}} to='/portfolio'>Portfolio</Link></LinkStyle>
+                <LinkStyle><LinkWrapper to='/portfolio'>Portfolio</LinkWrapper></LinkStyle>
```
Comment (yoloOnTheBattlefield) [inline src/components/navComponent/NavComponent.js:13]: remove LinkStyle?

File: src/components/styles/Nav.styled.js [hunk 4]
```diff
@@ -13,6 +14,10 @@ export const LeftSide = styled.div`
     display: flex;
     flex-direction: row;
     justify-content: center;
+    :LinK{
+        text-decoration: 'none';
+        color:'white';
+    }
```
Comment (yoloOnTheBattlefield) [inline src/components/styles/Nav.styled.js:20]: remove this

File: src/pages/Home/index.js [hunk 5]
```diff
@@ -1,11 +1,34 @@
- import NavComponent from "../../components/navComponent/NavComponent"
-
- const Home = () => {
-    return(
-        <div>
-            <NavComponent/>
-        </div>
-    )
+ import React from "react"
+ import axios from 'axios'
```
Comment (yoloOnTheBattlefield) [inline src/pages/Home/index.js:2]: use same quotes

File: src/pages/Home/index.js [hunk 6]
```diff
@@ -1,11 +1,34 @@
...
+    state={
+        list: null,
+        isLoading: false
+    }
+
+    getData = async () => {
+        this.setState({isLoading: true})
+        const response = await axios('https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=50&page=1&sparkline=true&price_change_percentage=1h%2C24h%2C7d')
+        const data = response.data
+        this.setState({list: data, isLoading: false})
```
Comment (yoloOnTheBattlefield) [inline src/pages/Home/index.js:14]: this.setState({ list: data, isLoading: false })

File: src/pages/Home/index.js [hunk 7]
```diff
@@ -1,11 +1,34 @@
...
+ class Home extends React.Component{
+    state={
+        list: null,
+        isLoading: false
+    }
+
+    getData = async () => {
+        this.setState({isLoading: true})
+        const response = await axios('https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=50&page=1&sparkline=true&price_change_percentage=1h%2C24h%2C7d')
+        const data = response.data
```
Comment (yoloOnTheBattlefield) [inline src/pages/Home/index.js:13]: const { data} = await axios('https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=50&page=1&sparkline=true&price_change_percentage=1h%2C24h%2C7d')

File: src/pages/Home/index.js [hunk 8]
```diff
@@ -1,11 +1,34 @@
...
+        this.setState({list: data, isLoading: false})
+    }
+
+    componentDidMount(){
+        this.getData()
+    }
+
+    render (){
```
Comment (yoloOnTheBattlefield) [inline src/pages/Home/index.js:21]: render()

File: src/pages/coin/index.js [hunk 9]
```diff
@@ -1,6 +1,7 @@
 
 const Coin = () => {
-    return <h2>Coins</h2>
+    return (
+    <h2>Coins</h2>)
```
Comment (yoloOnTheBattlefield) [inline src/pages/coin/index.js:4]: return (
    <h2>Coins</h2>
  );

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/pages/Home/index.js (hunk 8 above)
Comment (kabelic) [reply src/pages/Home/index.js:21]: done

File: src/pages/Home/index.js (hunk 7 above)
Comment (kabelic) [reply src/pages/Home/index.js:13]: done

File: src/pages/coin/index.js (hunk 9 above)
Comment (kabelic) [reply src/pages/coin/index.js:4]: done

File: src/pages/Home/index.js (hunk 6 above)
Comment (kabelic) [reply src/pages/Home/index.js:14]: done

File: src/pages/Home/index.js (hunk 5 above)
Comment (kabelic) [reply src/pages/Home/index.js:2]: done

File: src/components/navComponent/NavComponent.js (hunk 2 above)
Comment (kabelic) [reply src/components/navComponent/NavComponent.js:1]: done

File: src/components/navComponent/NavComponent.js (hunk 3 above)
Comment (kabelic) [reply src/components/navComponent/NavComponent.js:13]: done

File: src/components/LinkComponent.js (hunk 1 above)
Comment (kabelic) [reply src/components/LinkComponent.js:5]: done

**Pull request merged**